src/controllers/confirmation.controller.ts

```diff
@@ -4,7 +4,7 @@ import { logger } from "../utils/logger";
 import { CONFIRMATION_PAGE } from "../config";
 import { getLoggedInUserEmail } from "../utils/session";
 import { getApplicationData } from "../utils/application.data";
-import { ApplicationData } from "../model/application.model";
+import { ApplicationData } from "../model";
```

Comment:
i think the import will still work as ApplicationData is exported from model/index.ts but seems out of step with our other imports. Is it ok to change this import back?

Reply:
Sure, it's just something my IDE flagged to simplify it. I'll turn it back